enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/HighLimit.java

```diff
@@ -54,7 +54,7 @@
 
     public HighLimit()
     {
-        super( STORE_VERSION, 7, Capability.DENSE_NODES, Capability.SCHEMA, Capability.LUCENE_5 );
+        super( STORE_VERSION, 8, Capability.DENSE_NODES, Capability.SCHEMA, Capability.LUCENE_5 );
```

Comment:
We can't require a migration until 3.1. I just notice that the old format code is still around but not hooked up to anything. I guess you plan to use it to provide migration paths in 3.1?